main/webapp/modules/core/scripts/facets/text-search-facet.js

```diff
@@ -147,7 +147,8 @@ TextSearchFacet.prototype._initializeUI = function() {
       return;
     }
     self._query = this.value;
-    self._scheduleUpdate();
+    setTimeout(function(){ self._scheduleUpdate(); }, 1500);
```

Comment:
1200 and 1300 feel a bit more natural to me, whereas 1500+ feels awkward. But then, the original issue was that of Regex entry into the Text Filter box, and when I test with a bit of Regex, indeed, 1500 is a good balance. Let's go with that!

Reply:
So it's good to go then ?

Reply:
We'll let a few other folks who use this often and see what they think.  It seems to work fine also and delay with the Regex error propagation also for instance if doing a `(a)(b)`

@ostephens @ettorerizza How does this feel for you when typing Regex?  too long? too short?